spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user1) { User.new(name: 'Mulugeta M', email: 'mulie@gmail.com', password: '12345678') }
+
+  context 'validation' do
+    it 'user has name' do
```

Comment:
- [ ] A spec test case is failing! Please let's ensure all test cases are passing!

**Failing test case:**
![image](https://user-images.githubusercontent.com/12745474/221381668-a8f5c541-fa71-446a-bb7b-e64d2fe7f1c9.png)